src/index.html

```diff
@@ -11,7 +11,193 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend >Personal information</legend>
+        <label class="form-label">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+            autocomplete="off"
+          >
+        </label>
+        <label class="form-label">
+          Name:
+          <input
+            type="text"
+            name="name"
+            id="name"
+            autocomplete="off"
+            minlength="3"
+            maxlength="10"
+          >
+        </label>
+        <label class="form-label">
+          How old are You?
+          <input
+            type="number"
+            name="years-old"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+        <label class="form-label">
+          Full date of birth:
+          <input
+            type="date"
+            name="dateOfBirth"
+            min="1923-01-01"
+            max="2022-01-01"
+          >
+        </label>
+        <label>
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <label class="form-label">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+        <label class="form-label2">
+          Password:
+          <input
+            type="password"
+            name="password"
+            id="password"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <label class="form-label">
```

Comment:
shouldn't be `label` here bc on click on `Do you love cats?` it selects first option `Yes`